x-pack/plugins/fleet/public/hooks/use_request/epm.ts

```diff
@@ -270,14 +270,11 @@ export const sendBulkInstallPackages = (
   });
 };
 
-export const sendRemovePackage = (pkgName: string, pkgVersion: string, force: boolean = false) => {
+export const sendRemovePackage = (pkgName: string, pkgVersion: string) => {
```

Comment:
I took a look at the code and I think we should move the force flag from `body` to `query`. I believe `DeletePackageRequestSchema` is the schema enforce on this route, not `DeleteKibanaAssetsRequestSchema`

Reply:
Thanks Jen, you were totally right! I made this change on a rush on Friday evening and got the schemas mixed up. I fixed it, it should be correct now.